content/shared/influxdb3-sample-data/sample-data.md

```diff
@@ -374,6 +433,16 @@ influxdb3 write \
 ```
 {{% /code-placeholders %}}
 
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+
+{{% code-placeholders "AUTH_TOKEN|DATABASE_NAME" %}}
+```sh
+curl -v "http://localhost:8181/api/v3/write_lp?db=sensors&precision=auto&accept_partial=false" \
+  --data-raw "$(curl --request GET https://docs.influxdata.com/downloads/bay-area-weather.lp)"
```

Comment:
Did you test this? Does it need to use `--data-binary` instead, bc it's downloading the `.lp` first and then posting the file?